static/app/utils/metrics.tsx

```diff
@@ -61,9 +63,16 @@ export function useMetricsTagValues(mri: string, tag: string) {
   );
 }
 
+type DateTime = {
+  end: DateString | null;
+  period: string | null;
+  start: DateString | null;
+  utc: boolean | null;
+};
```

Comment:
Do we need to define this type? Sounds like something we would already have type for. Maybe PageFilters['datetime']?